README.md

```diff
@@ -1,15 +1,14 @@
-Pelux reference manifests
+PELUX baseline manifests
 =========================
-This is the manifest repository for building the Pelux reference for various targets using Yocto 
+This is the manifest repository for building the PELUX reference for various targets using Yocto
```

Comment:
Should it say "baseline" here as well?

What "targets" mean here should be made more clear. I assume this means NUC or Minnowboard right now?